### PR TITLE
CIP 0012: More granular validator signer keys

### PR DESCRIPTION
**Pull request by kobigurk**

### CODE REVIEW

File: CIPs/0012.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# CIP 0012: More granular validator signer keys
+
+- Date: 2020-05-20
+- Author: @kobigurk
+- Status: WIP
+
+## Overview
+
+Validator signer keys allow to both sign consensus blocks and rotate BLS keys. We suggest to add more granularity to support the following use case: ECDSA signing of consensus messages without the ability to rotate BLS keys. This would be useful for situations where the BLS key is well protected while the ECDSA key isn't. Since the BLS key signs the COMMIT messages, the danger rests mostly there anyway. Examples of these kinds of situations are threshold signatures for BLS and Ledger signing for BLS.
```
Comment (nambrot) [inline CIPs/0012.md:9]: What are the situations in which the BLS key is better protected than the ECDSA key?

Comment (kobigurk) [reply CIPs/0012.md]: Expanded the description to answer it :)

File: CIPs/0012.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# CIP 0012: More granular validator signer keys
+
+- Date: 2020-05-20
+- Author: @kobigurk
+- Status: WIP
```
Comment (YazzyYaz) [inline CIPs/0012.md:5]: hi @kobigurk  thank you for submitting this CIP, I would like you to update it to follow the newly activated CIP process which has it's newly updated template here: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md

Also, if the CIP is in Draft status, I'd be able to merge the proposal into the repo.

**Comment by kobigurk** [comment]
Hey @YazzyYaz, thanks, for the time being this doesn't seem like a priority, so I'm going to close it.

**Pull request closed without merging**